lifter/utils.cpp

```diff
@@ -47,6 +47,12 @@ uintptr_t address_to_mapped_address(void* fileBase, uintptr_t rva) {
 
 namespace debugging {
 
+    llvm::raw_ostream& operator<<(llvm::raw_ostream& OS,
+                                  const llvm::KnownBits& KB) {
+        KB.print(OS);
+        return OS;
+    }
+
```

Comment:
Starting from LLVM version 17 and above this isnt needed. To prevent the compiler from throwing ambiguity errors, we can wrap this in the LLVM_VERSION_MAJOR macro for versions above 17.

Reply:
ok, added